docs/modules/ROOT/pages/luna-for-ragstack.adoc

```diff
@@ -5,9 +5,10 @@ A subscription to {company} Luna for RAGStack gives you direct access to *suppor
 
 == Advantages of Luna for RAGStack
 
-RAGStack is an out of the box solution simplifying RAG in GenAI apps. RAGStack includes a RAG implementation, giving you a comprehensive GenAI Stack that leverages xref:examples:index.adoc#langchain-astraLangChain, xref:examples:index.adoc#llama-astraLlamaIndex, and more -- with expert support from {company}.
+RAGStack is an out of the box solution simplifying RAG in GenAI apps. RAGStack includes a RAG implementation, giving you a comprehensive GenAI Stack that leverages xref:examples:index.adoc#langchain-astraLangChain[
+LangChain and Astra DB Serverless], xref:examples:index.adoc#llama-astraLlamaIndex[LlamaIndex and Astra DB Serverless], and more -- with expert support from {company}.
 
-Adding external components to a GenAI platform -- to support development and production environments -- can be complex. With Luna for RAGStack, you can confidently implement your own GenAI platform, and deploy the RAGStack open-source framework into production, with the peace of mind that comes from having direct access to experts.
+Adding external components to a GenAI platform -- to support development and production environments -- can be complex. With the *core RAGStack functionality* supported by Luna for RAGStack, you can confidently implement your own GenAI platform, and deploy the RAGStack open-source framework into production, with the peace of mind that comes from having direct access to experts.
```

Comment:
- The dashes are not necessary. 
- Avoid passive voice (supported by).
- You don't need all those commas.

Reply:
Thanks - will fix!